docs/scos/dev/front-end-development/202212.0/oryx/architecture/reactivity/reactive-components.md

```diff
@@ -43,37 +45,34 @@ export class ProductPriceComponent {
 
 ## Updating data in the DOM
 
-While the user navigates thorugh a single page application, it's crucial for components to be updated accordingly. This section describes how data updates are propagated throughout the UI in Oryx.
+While the user navigates thorugh a single page application, it is crucial for components to be updated accordingly. This section describes how data updates are propagated throughout the UI in Oryx.
 
 RxJS operates on data streams and updates them in memory, but it doesn't synchronize this to the UI automatically. Each JavaScript framework ships its own opinionated method to update the DOM. The selected method contributes significantly to the performance and user experience of the application.
 
-The components provided in the Oryx libraries are built with Lit. Lit provides a highly efficient system to only synchronize the minimum required updates to the DOM. When updates are loaded asynchronously, the UI needs to be updated every time new data is emitted. To make this as transparent as possible, the `@signalAware()` decorator is used in the UI components, allowing for `signals` to be used when building UI, which automatically requests updates to the view when needed.
+The components provided in the Oryx libraries are built with Lit. Lit provides a highly efficient system to only synchronize the minimum required updates to the DOM. When updates are loaded asynchronously, the UI needs to be updated every time new data is emitted. To make this as transparent as possible, the `@asyncState()` decorator is used in the UI components. Under the hood, the decorator uses `AsyncStateController` which requests updates to the view when needed.
+
+The following example shows the usage of the `asyncState` decorator. The decorator subscribes to the assigned observable and requests an update to the component when needed. This means that component developers do not need to worry about how the reactive system works under the hood.
 
-The following example shows the usage of the `computed` signal. `Computed` wraps an observable into a `signal` that automatically subscribes to the underlying observable and triggers updating the view. This means that component developers don't need to worry about how the reactive system works under the hood.
+Oryx components are built in TypeScript, and we provide types everywhere to improve developer experience and avoid errors upfront. The original type of the assigned observable needs to be adjusted. It's impossible to resolve a correct type from the observable by using a decorator, which is why the `valueType` function is used to resolve the observed type.
```

Comment:
Please convert the second sentence into active voice. Make it clear as to who's supposed to adjust it.